haskell/src/Lorentz/Contracts/Spec/FA2Interface.hs

```diff
@@ -17,7 +17,8 @@ module Lorentz.Contracts.Spec.FA2Interface
   , OperatorTransferMode (..)
   , OwnerTransferMode (..)
   , Parameter (..)
-  , ParameterC
+  , FA2ParameterC
+  , FA2ParameterMichelsonEntrypoints
```

Comment:
We are having the FA2 stuff here as a temporary arrangement, and we will eventually import all FA2 interface stuff/tests from Morley. So may be it is better not to depend on specific changes to this module.